packages/cozy-client/src/assets/qualifications.json

```diff
@@ -491,6 +491,9 @@
     {
       "label": "other_administrative_document"
     },
+    {
+      "label": "expense_claim"
+    },
```

Comment:
Would you mind just adding the purpose? :pray: 
```
{
  "label": "expense_claim"
  "purpose": "invoice"  
},
```

Reply:
pas de souci, avec @BenjaminMty on ne savait pas trop quoi mettre 👍